run_tests

```diff
@@ -24,12 +28,19 @@ else
   SKIP_CONDA=1
 fi
 
+# read command-line arguments
 ARGS=()
 for A in "$@"; do
     case $A in
         --skip-conda)
             SKIP_CONDA=0
             ;;
+        --plugins)
+            TEST_SET=1
+            ;;
+        --test_all)
+            TEST_SET=2
```

Comment:
all should be the default

Reply:
done, but this requires a change in CIVET not to run the plugins as required. PR is pending.